tests/communications/test_dto_serialization.py

```diff
@@ -85,9 +84,13 @@
     "TestSuiteParam": ["projectKey"],
     "WeightsPerFeature": [],
     "WorkerReply": [],
+    "AbortParams": ["job_id"],
```

Comment:
Could you use the code at the bottom of this file to generate the changes ? 
Items are sorted by alphabetical order here I think

Reply:
done